src/hidimstat/distilled_conditional_randomization_test.py

```diff
@@ -346,18 +315,19 @@ def importance(
         else:
             X_ = X
         n_samples, n_features = X_.shape
-        selection_features = np.arange(n_features)[np.logical_not(self.non_selection_)]
+        selection_features = np.arange(n_features)[self.selection_set]
```

Comment:
```suggestion
```
It's not necessary to create a variable for it.

Reply:
It's just convenient since it's used twice L323 and L371